### PR TITLE
BI-10288 Extract individual pscs and display

### PR DESCRIPTION
**Pull request by mattch1**
Updated template to use existing includes for individual psc and to add the other types.
Updated controller and based on the work completed for the active officer controller, reusing the existing formatting.

### CODE REVIEW

File: src/controllers/tasks/active.psc.details.controller.ts [hunk 1]
```diff
@@ -2,14 +2,57 @@ import { NextFunction, Request, Response } from "express";
...
 import { urlUtils } from "../../utils/url";
+import { PersonOfSignificantControl } from "@companieshouse/api-sdk-node/dist/services/confirmation-statement";
+import { getPscs } from "../../services/psc.service";
+import { Session } from "@companieshouse/node-session-handler";
+import { appointmentTypes, LOCALE_EN } from "../../utils/constants";
+import { formatAddressForDisplay, formatPSCForDisplay } from "../../utils/format";
+import { toReadableFormat } from "../../utils/date";
 
-export const get = (req: Request, res: Response, next: NextFunction) => {
+
```
Comment (SVarallo) [inline src/controllers/tasks/active.psc.details.controller.ts:12]: Don't really need these extra spaces

File: views/tasks/active-psc-details.html [hunk 2]
```diff
@@ -177,10 +118,10 @@ <h2 class="govuk-heading-m">1 other registrable person</h2>
             <div class="govuk-summary-card govuk-!-margin-bottom-4">
               <div class="govuk-summary-card_heading govuk-!-padding-top-4 govuk-!-padding-bottom-4 govuk-!-padding-right-3 govuk-!-padding-left-3" style="display: flex; justify-content: space-between; align-items: center">
                 <p class="govuk-!-font-weight-bold govuk-!-margin-0">AYR BAPTIST CHURCH</p>
-                 {{govukTag({
-                  text: "Active",
-                  classes: "govuk-tag--green"
-                 })}}
+                {{govukTag({
+                text: "Active",
+                classes: "govuk-tag--green"
```
Comment (SVarallo) [inline views/tasks/active-psc-details.html:123]: How come you removed the indentation in this tag and the one above it?

File: src/controllers/tasks/active.psc.details.controller.ts [hunk 3]
```diff
@@ -2,14 +2,57 @@ import { NextFunction, Request, Response } from "express";
...
+        formattedPsc: formattedPsc,
+        ura: ura,
+        serviceAddress: serviceAddress,
+        dob: dob,
+        dateOfAppointment: dateOfAppointment
+      };
+    });
+};
+
+const equalsIgnoreCase = (pscType: string, wantedPscType: string): boolean => {
```
Comment (mwestacott) [inline src/controllers/tasks/active.psc.details.controller.ts:56]: Guessing you copied this from the previous implementation we did for officers? Could we maybe move the function somewhere more central, rename the parameters to something more general and avoid the duplication of code? Then just call it from the two controllers...

File: views/tasks/active-psc-details.html (hunk 2 above)
Comment (mattch1) [reply views/tasks/active-psc-details.html:123]: This code is due to come out in the other subtasks.

**Review by mwestacott — APPROVED** [review]

**Review by SVarallo — APPROVED** [review]

**Pull request merged**